src/Command/RunCommand.php

```diff
@@ -17,6 +18,19 @@
  */
 class RunCommand extends Command {
 
+	/**
+	 * @var \Cake\Core\ContainerInterface
+	 */
+	protected $container;
+
+	/**
+	 * @param \Cake\Core\ContainerInterface $container
+	 */
+	public function __construct(ContainerInterface $container){
```

Comment:
```suggestion
	public function __construct(ContainerInterface $container) {
```
I am still confused why CS is not working (anymore)